oml/metrics/embeddings.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         cmc_top_k: Tuple[int, ...] = (5,),
         precision_top_k: Tuple[int, ...] = (5,),
         map_top_k: Tuple[int, ...] = (5,),
-        fmr_vals: Tuple[float, ...] = tuple(),
+        fmr_vals: Tuple[float, ...] = None,
```

Comment:
we don't to change this line, it was already empty
so, roll back this particular line please

Reply:
done